CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ release.
 - Projection information (through GDAL) will be attached to the ISD if a projected product is processed through ALE [#524](https://github.com/DOI-USGS/ale/pull/524)
 - Kaguya IsisLabelNaifSpice driver, tests, and test data [#487](https://github.com/DOI-USGS/ale/pull/487)
 - Hayabusa Amica IsisLabelNaifSpice driver, tests and test data [#521](https://github.com/DOI-USGS/ale/pull/521)
+- Apollo Metric drivers, tests, and data [#533](https://github.com/DOI-USGS/ale/pull/533)
```

Comment:
@AustinSanders Could this be moved out from the 0.9.0 section? This was a result of the various releases that came through while this PR was open